rust/gui-client/src-tauri/src/client/wintun_install.rs

```diff
@@ -25,8 +25,6 @@ pub(crate) enum Error {
     DllPathInvalid,
     #[error("permission denied")]
     PermissionDenied,
-    #[error("platform not supported")]
-    PlatformNotSupported,
```

Comment:
Making this a runtime error seems unnecessary. Compilation will already fail for anything other than x64 and aarch64 because we don't have a fallback version of `get_platform_dll_bytes`.